constraints/miden-vm/memory.air

```diff
@@ -0,0 +1,101 @@
+mod MemoryChipletAir
+
+### Helper functions/evaluators ###################################################################
+
+fn delta(v: scalar) -> scalar:
+    return v' - v
+
+
+# Enforces that columns must be zero.
```

Comment:
nit: Should we change this to `Enforces that the provided column must be zero.`?

Reply:
I agree, that sounds better. But should we say `columns` instead of `column`, since we are checking 4 columns at once?

Reply:
Yeah, it should be `columns`.